lib/commons/dom/is-visible.js

```diff
@@ -58,6 +58,8 @@ dom.isVisible = function(el, screenReader, recursed) {
 		style.getPropertyValue('display') === 'none' ||
 		nodeName.toUpperCase() === 'STYLE' ||
 		nodeName.toUpperCase() === 'SCRIPT' ||
+		nodeName.toUpperCase() === 'NOSCRIPT' ||
+		nodeName.toUpperCase() === 'TEMPLATE' ||
```

Comment:
I'd suggest an `array.includes` solution instead of adding even more "or" cases.